File: back/DevC/src/main/java/com/back/devc/domain/member/member/dto/MyInfoResponse.java
package com.back.devc.domain.member.member.dto;

import com.back.devc.domain.member.member.entity.MemberRole;
import com.back.devc.domain.member.member.entity.MemberStatus;

import java.time.LocalDateTime;

public record MyInfoResponse(
        Long userId,
        String email,
        String nickname,
        MemberRole role,
        MemberStatus status,
        LocalDateTime createdAt
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/member/member/dto/MyInfoResponse.kt
package com.back.devc.domain.member.member.dto

import com.back.devc.domain.member.member.entity.MemberRole
import com.back.devc.domain.member.member.entity.MemberStatus
import java.time.LocalDateTime

data class MyInfoResponse(
    val userId: Long,
    val email: String,
    val nickname: String,
    val role: MemberRole,
    val status: MemberStatus,
    val createdAt: LocalDateTime
)
